app/build.gradle.kts

```diff
@@ -184,7 +178,7 @@ dependencies {
     implementation("androidx.compose.material:material-icons-extended")
 
     implementation("org.ocpsoft.prettytime:prettytime:5.0.8.Final")
-    implementation("androidx.navigation:navigation-compose")
+    implementation("androidx.navigation:navigation-compose:2.7.7")
```

Comment:
This version should come automatically with bom, no?

Reply:
It did not for me, must ve come from accompanist

Reply:
Gotcha.